---
# Documentation: https://wowchemy.com/docs/managing-content/

title: MetaEuk-sensitive, high-throughput gene discovery, and annotation for large-scale
  eukaryotic metagenomics.
subtitle: ''
summary: ''
authors:
- Eli Levy Karin
- Milot Mirdita
- Johannes SÃ¶ding
tags:
- Annotation
- Contigs
- Eukaryotes
- Homology detection
- Marine
- MetaEuk
- Prediction
categories: []
date: '2020-12-01'
lastmod: 2023-03-07T22:49:55+09:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2023-03-07T13:49:54.949309Z'
publication_types:
- '2'
abstract: BACKGROUND Metagenomics is revolutionizing the study of microorganisms and
  their involvement in biological, biomedical, and geochemical processes, allowing
  us to investigate by direct sequencing a tremendous diversity of organisms without
  the need for prior cultivation. Unicellular eukaryotes play essential roles in most
  microbial communities as chief predators, decomposers, phototrophs, bacterial hosts,
  symbionts, and parasites to plants and animals. Investigating their roles is therefore
  of great interest to ecology, biotechnology, human health, and evolution. However,
  the generally lower sequencing coverage, their more complex gene and genome architectures,
  and a lack of eukaryote-specific experimental and computational procedures have
  kept them on the sidelines of metagenomics. RESULTS MetaEuk is a toolkit for high-throughput,
  reference-based discovery, and annotation of protein-coding genes in eukaryotic
  metagenomic contigs. It performs fast searches with 6-frame-translated fragments
  covering all possible exons and optimally combines matches into multi-exon proteins.
  We used a benchmark of seven diverse, annotated genomes to show that MetaEuk is
  highly sensitive even under conditions of low sequence similarity to the reference
  database. To demonstrate MetaEuk's power to discover novel eukaryotic proteins in
  large-scale metagenomic data, we assembled contigs from 912 samples of the Tara
  Oceans project. MetaEuk predicted textgreater12,000,000 protein-coding genes in
  8 days on ten 16-core servers. Most of the discovered proteins are highly diverged
  from known proteins and originate from very sparsely sampled eukaryotic supergroups.
  CONCLUSION The open-source (GPLv3) MetaEuk software (https://github.com/soedinglab/metaeuk)
  enables large-scale eukaryotic metagenomics through reference-based, sensitive taxonomic
  and functional annotation. Video abstract.
publication: '*Microbiome*'
doi: 10.1186/s40168-020-00808-x
links:
- name: URL
  url: http://www.ncbi.nlm.nih.gov/pubmed/32245390 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC7126354
---
