---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Clustering predicted structures at the scale of the known protein universe
subtitle: ''
summary: ''
authors:
- Inigo Barrio-Hernandez
- Jingi Yeo
- Jürgen Jänes
- Milot Mirdita
- Cameron L. M. Gilchrist
- Tanita Wein
- Mihaly Varadi
- Sameer Velankar
- Pedro Beltrao
- Martin Steinegger
tags:
- Protein analysis
- Software
- Structural biology
categories: []
date: '2023-10-01'
lastmod: 2025-10-14T03:34:27+09:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2025-10-13T18:34:27.417186Z'
publication_types:
- '2'
abstract: Proteins are key to all cellular processes and their structure is important
  in understanding their function and evolution. Sequence-based predictions of protein
  structures have increased in accuracy, and over 214 million predicted structures
  are available in the AlphaFold database. However, studying protein structures at
  this scale requires highly efficient methods. Here, we developed a structural-alignment-based
  clustering algorithm—Foldseek cluster—that can cluster hundreds of millions of structures.
  Using this method, we have clustered all of the structures in the AlphaFold database,
  identifying 2.30 million non-singleton structural clusters, of which 31% lack annotations
  representing probable previously undescribed structures. Clusters without annotation
  tend to have few representatives covering only 4% of all proteins in the AlphaFold
  database. Evolutionary analysis suggests that most clusters are ancient in origin
  but 4% seem to be species specific, representing lower-quality predictions or examples
  of de novo gene birth. We also show how structural comparisons can be used to predict
  domain families and their relationships, identifying examples of remote structural
  similarity. On the basis of these analyses, we identify several examples of human
  immune-related proteins with putative remote homology in prokaryotic species, illustrating
  the value of this resource for studying protein function and evolution across the
  tree of life.
publication: '*Nature*'
doi: 10.1038/s41586-023-06510-w
links:
- name: URL
  url: https://www.nature.com/articles/s41586-023-06510-w
---
