---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Fast and sensitive taxonomic assignment to metagenomic contigs
subtitle: ''
summary: ''
authors:
- M Mirdita
- M Steinegger
- F Breitwieser
- J Söding
- E Levy Karin
tags: []
categories: []
date: '2021-09-01'
lastmod: 2025-10-14T03:34:30+09:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2025-10-13T18:34:30.305042Z'
publication_types:
- '2'
abstract: MMseqs2 taxonomy is a new tool to assign taxonomic labels to metagenomic
  contigs. It extracts all possible protein fragments from each contig, quickly retains
  those that can contribute to taxonomic annotation, assigns them with robust labels
  and determines the contig’s taxonomic identity by weighted voting. Its fragment
  extraction step is suitable for the analysis of all domains of life. MMseqs2 taxonomy
  is 2–18× faster than state-of-the-art tools and also contains new modules for creating
  and manipulating taxonomic reference databases as well as reporting and visualizing
  taxonomic assignments. Availability and implementation MMseqs2 taxonomy is part
  of the MMseqs2 free open-source software package available for Linux, macOS and
  Windows at https://mmseqs.com.
publication: '*Bioinformatics*'
doi: 10.1093/bioinformatics/btab184
links:
- name: URL
  url: https://academic.oup.com/bioinformatics/article/37/18/3029/6178277
---
